.buildkite/pipeline.testing.yml

```diff
@@ -20,6 +20,23 @@ steps:
             .buildkite/scripts/grapl_testing_stack/dump_artifacts.sh
     command:
       - .buildkite/scripts/grapl_testing_stack/run_parameterized_job.sh typescript-integration-tests 6
+    artifact_paths:
+      - "test_artifacts/**/*"
+
+  - wait
+
+  - label: ":aws: Integration tests in AWS"
```

Comment:
I think you'll need to give the steps different names (e.g. "Rust Integration tests in AWS", "TypeScript Integration tests in AWS")